pinot-core/src/main/java/org/apache/pinot/core/segment/index/datasource/BaseDataSource.java

```diff
@@ -44,13 +44,10 @@
   private final NullValueVectorReader _nullValueVector;
 
   public BaseDataSource(DataSourceMetadata dataSourceMetadata, ForwardIndexReader<?> forwardIndex,
-                        @Nullable Dictionary dictionary, @Nullable InvertedIndexReader<?> invertedIndex,
-                        @Nullable InvertedIndexReader<?> rangeIndex,
-                        @Nullable H3IndexReader h3Index,
-                        @Nullable TextIndexReader textIndex,
-                        @Nullable TextIndexReader fstIndex,
-                        @Nullable BloomFilterReader bloomFilter,
-                        @Nullable NullValueVectorReader nullValueVector) {
+      @Nullable Dictionary dictionary, @Nullable InvertedIndexReader<?> invertedIndex,
```

Comment:
make sure you are using the stylesheet to format the code

Reply:
Yes, this is done via auto-formatting.